### PR TITLE
Testing To Do list: part 1

### PR DESCRIPTION
**Pull request by SlowKingV**
## Description:
In this project, you will write unit tests for the To Do list application. The testing project has been divided into two parts. In part 1 you will test functions responsible for adding and deleting list items. In part 2 you will test the status and content updates as well as the order re-position of items in the list.

### Project Requirements:
- This is a pair-programming project. Please decide whose project will you use for testing.
- Use jest framework for testing.
- Create a test file (`[..].test.js`) for a file containing the **add item** and **delete item**  functions that you want to test.
- Take a look at each of these functions. Are they *pure functions*? If the answer is "yes" then writing tests for them should be straightforward. Some of those functions however will update *localStorage* and manipulate the *DOM*. For those, you will need to use mocks:
  - Mock a storage object to "imitate" localStorage operations (only if you are using `jest < 24.x`, in newer versions localStorage is mocked automatically).
  - Mock HTML to test if add/delete functions add or remove exactly one `<li>` element to/from the list in the DOM.
- Make sure you group your tests using the `describe()` method.

### CODE REVIEW

File: src/modules/DOMFunctions.js [hunk 1]
```diff
@@ -34,14 +34,14 @@ export const tasklistAdd = (tasklist, value) => {
...
 // Set Handler for 'Clear All Completed' button
 export const clearAll = (tasklist) => {
   tasklist.clearAllCompleted();
   tasklist.updateListHTML();
   updateLocalStorage(tasklist);
-}
+};
 
 export const loadData = (tasklist) => {
   if (localStorage.datalist && localStorage.datalist.length > 0) {
```
Comment (Ismailco) [inline src/modules/DOMFunctions.js:47]: - [x] Good job testing the `tasklistAdd` and the DOM remove button, however, you should create tests for all the pure functions responsible for adding and also deleting list items, that includes the clearAll remove tasks

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @SlowKingV, @ijeomaodinko, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/DOMFunctions.js (hunk 1 above)
Comment (SlowKingV) [reply src/modules/DOMFunctions.js:47]: We have added a test for the `clearAll()` function.

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>

 **Hello Team**,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**